tests/unit/distance/EuclideanDistance_unittest.cc

```diff
@@ -148,3 +148,26 @@ TEST(EuclideanDistance, heterogenous_features)
 	SG_UNREF(features_lhs);
 	SG_UNREF(features_rhs);
 }
+
+TEST(EuclideanDistance, sparse_features)
+{
+	CDenseFeatures<float64_t>* features_lhs=create_lhs();
```

Comment:
Can you test this with a really high dimensional and really sparse set of vectors?
To make sure it doesnt blow up memory?
Like 10mio dims with 100 components nonzero